media-video/mpv/mpv-9999.ebuild

```diff
@@ -150,8 +148,7 @@ pkg_setup() {
 }
 
 src_prepare() {
-	cp "${DISTDIR}"/waf-${WAF_V} "${S}"/waf || die
-	chmod 0755 "${S}"/waf || die
+	python bootstrap.py || die
```

Comment:
This is a bad idea -- bootstrap.py script downloads WAF from internet. I don't think we allow this in ebuilds. Moreover, it won't work with ``FEATURES="network-sandbox"``.